Xamarin.PropertyEditing.Mac/Controls/CharEditorControl.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Collections;
+using AppKit;
+using Xamarin.PropertyEditing.Mac.Resources;
+using Xamarin.PropertyEditing.ViewModels;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	internal class CharEditorControl : PropertyEditorControl<PropertyViewModel<char>>
+	{
+		public CharEditorControl ()
+		{
+			StringEditor = new CharTextField {
+				BackgroundColor = NSColor.Clear,
```

Comment:
Sorry, I just copied the current implementation in StringEditorControl
https://github.com/xamarin/Xamarin.PropertyEditing/blob/701fe5ee0b619bde33d11f1230496bc925fdd0a1/Xamarin.PropertyEditing.Mac/Controls/StringEditorControl.cs#L15-L21

Maybe it could be interesting think a way to reuse this XXTextField into other EditorControls, like create a ValidatedTextField

Reply:
In the `StringEditorControl` I'm using the base class `NSTextField` so I essentially have to override various properties, while yours is a custom class, hence why I asked if you envisioned it being reused.

A `ValidatedTextField` might be a good idea, if designed correctly. Maybe add it as an internal feature request and Eric can take a look?